### ISSUE TITLE
sameSite cookie behavior set to change soon in Firefox

### ISSUE THREAD

**Issue by ungless** [PostHog/posthog-js#76]
This could be potentially (I think?) breaking for PostHog on Firefox so we should do something about this.

- See https://hacks.mozilla.org/2020/08/changes-to-samesite-cookie-behavior/

- Also see https://github.com/PostHog/posthog/issues/1207 for additional discussion.

- This may also be an issue with other integrations, but for now @mariusandra pointed out posthog-js in particular.

**Comment by mariusandra** [comment]
It's just with the JS integrations, but it's in other browsers as well. For now those are just warnings, but we need to fix them as the next versions of the browsers (due whenever) might prevent posthog-js from working.

**Comment by mariusandra** [comment]
Also: https://github.com/PostHog/posthog-js/issues/17

**Comment by mariusandra** [comment]
This should be fixed with `1.3.8` (released 3 days ago). The "ph_" cookie now has the "secure" flag set if it was set under a "https" url:

![image](https://user-images.githubusercontent.com/53387/89776048-642bc480-db09-11ea-970a-6d6f61f21274.png)

**Comment by toger5** [comment]
With firefox 109 I am getting the following warning:
```
Some cookies are misusing the recommended “SameSite“ attribute 4

Cookie “ph_phc_rXGHx9vDmyEvyRxPziYtdVIv0ahEv8A9uLWFcCi1WcU_posthog” does not have a proper “SameSite” attribute value. Soon, cookies without the “SameSite” attribute or with an invalid value will be treated as “Lax”. This means that the cookie will no longer be sent in third-party contexts. If your application depends on this cookie being available in such contexts, please add the “SameSite=None“ attribute to it. To know more about the “SameSite“ attribute, read https://developer.mozilla.org/docs/Web/HTTP/Headers/Set-Cookie/SameSite
```
And posthog does not seem to send events anymore (It worked before. has there been some changes recently?)
